src/Exceptions/HttpBadRequest.php

```diff
@@ -8,4 +8,3 @@ class HttpBadRequest extends HttpException {
     public const DESCRIPTION = "Indicates that the server cannot or will not process the request because the received syntax is invalid, nonsensical, or exceeds some limitation on what the server is willing to process.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces